### PR TITLE
Term 20

### PR DESCRIPTION
**Pull request by chrismytton**
This adds term 20 information from [this page](https://ko.wikipedia.org/wiki/%EB%8C%80%ED%95%9C%EB%AF%BC%EA%B5%AD_%EC%A0%9C20%EB%8C%80_%EA%B5%AD%ED%9A%8C%EC%9D%98%EC%9B%90_%EB%AA%A9%EB%A1%9D).

Part of https://github.com/everypolitician/everypolitician-data/issues/18448

### CODE REVIEW

File: scraper.rb [hunk 1]
```diff
@@ -4,13 +4,19 @@
...
-EveryPolitician::Wikidata.scrape_wikidata(names: { ko: names }, output: false)
-warn EveryPolitician::Wikidata.notify_rebuilder
+names_20 = EveryPolitician::Wikidata.wikipedia_xpath(
+  url: 'https://ko.wikipedia.org/wiki/대한민국_제20대_국회의원_목록',
+  after: "//h2/span[.='가나다순 보기']",
+  before: "//h2/span[.='의석 변동']",
+  xpath: '//p//a[not(@class="new")]/@title',
+)
 
+EveryPolitician::Wikidata.scrape_wikidata(names: { ko: names_19 + names_20 }, output: false)
```
Comment (tmtmtmtm) [inline scraper.rb]: Tiniest of tiny nits, but probably a little better to make that `names_19 | names_20` as we expect there'll be some overlap there.

File: scraper.rb [hunk 2]
```diff
@@ -4,13 +4,19 @@
...
+warn EveryPolitician::Wikidata.notify_rebuilder
```
Comment (tmtmtmtm) [inline scraper.rb]: We should really replace this with a webhook call now

**Review by tmtmtmtm — CHANGES_REQUESTED** [review]

File: scraper.rb (hunk 1 above)
Comment (chrismytton) [reply scraper.rb]: Done in https://github.com/everypolitician-scrapers/korean-assembly-members-wikidata/pull/1/commits/4aaae59de5e3509dd13bec9719d9000620ff534f

File: scraper.rb (hunk 2 above)
Comment (chrismytton) [reply scraper.rb]: Good idea. I've removed the notifier code in https://github.com/everypolitician-scrapers/korean-assembly-members-wikidata/pull/1/commits/7dc85198aa22b717248964bbfb178b7f86ac8838. Can you update https://morph.io/tmtmtmtm/korean-assembly-members-wikidata and move the environment variable to be a webhook please?

**Comment by chrismytton** [comment]
@tmtmtmtm 👀

**Review by tmtmtmtm — APPROVED** [review]

File: scraper.rb (hunk 2 above)
Comment (tmtmtmtm) [reply scraper.rb]: Done, though the general pattern for scrapers that can be moved safely (e.g. Wikidata ones where we clean out the database on every run anyway) is to just make a new everypolitician-scrapers version and then remove the personal version.

**Comment by tmtmtmtm** [comment]
👍  — I've also triggered a new run of the scraper post-merge.

**Pull request merged**